sdk/src/main/java/io/dapr/client/domain/CloudEvent.java

```diff
@@ -76,6 +79,37 @@
   @JsonProperty("data_base64")
   private byte[] binaryData;
 
+  /**
+   * The pubsub component this CloudEvent came from.
+   */
+  private String pubsubname;

```

Comment:
camelCase on properties, please.

Reply:
Sure, I was just trying to follow the above that didn't have camelCase:
private String specversion;
private String datacontenttype;

Reply:
OK. We should fix those too.

Reply:
I've made them camelCase and included @JsonProperty("lowercase") for each of them as the Dapr CloudEvent uses lowercasenames and JSON/OBJECT_MAPPER are case-sensitive.